### DISCUSSION TITLE
树图不同层级节点不同布局

### DISCUSSION THREAD

**Discussion by mrdulin** [antvis/G6#6411] [Q&A]
如图，1，2级节点是至上而下紧凑树，2，3级节点是缩进树
![image](https://github.com/user-attachments/assets/b382161f-f6a4-4203-92d5-1ca2dd2a1b4e)

可否实现

**Comment by Aarebecca** [comment]
目前暂还不支持该种布局，可以考虑自定义布局、边实现

**Reply by mrdulin** [reply]
感谢回复，可否提供个简单的例子
